src/tictactoe/assets/js/api.js

```diff
@@ -0,0 +1,20 @@
+
+export function makePostRequest(url, body, onSuccess){
+    const settings = {
+        method: 'POST',
+        headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(body)
+    }
+    return fetch(url, settings)
+        .then(response => { return response.json(); })
+        .then(data => onSuccess(data))
```

Comment:
This is a topic to discuss. 
I made like this because these requests are used in `react-redux` methods like
https://github.com/SergiiVolodkoWorking/TicTacToeOnline/pull/5/files#diff-4219311fcbd337c03d7df9bd40bc3ad8
I'm implementing them for the first time, so was not sure how will they work with async/await